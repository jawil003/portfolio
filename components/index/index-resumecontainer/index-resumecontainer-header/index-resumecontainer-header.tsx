import React from "react";
import designSystem from "../../../../styles/designSystem";
import { Typography } from "../../../shared/typography";

export interface Props {}

/**
 * An ResumeHeader React Component.
 * @author Jannik Will
 * @version 0.1
 */
export const IndexResumeContainerHeader: React.FC<Props> = ({
  children,
}) => {
  return (
    <Typography
      variant="h4"
      bold="semi-bold"
      italic
      color={
        designSystem.colors.brand
          .secondaryText
      }
    >
      {children}
    </Typography>
  );
};
