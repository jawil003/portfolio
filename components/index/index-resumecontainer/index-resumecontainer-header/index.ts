export * from "./index-resumecontainer-header";
