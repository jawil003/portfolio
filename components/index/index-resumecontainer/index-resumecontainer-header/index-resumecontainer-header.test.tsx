import { render } from "@testing-library/react";
import { IndexResumeContainerHeader } from "./index-resumecontainer-header";

describe("IndexResumeContainerHeader", () => {
  test("Create IndexResumeContainerHeader", () => {
    const testHeader = render(
      <IndexResumeContainerHeader>
        Hello World
      </IndexResumeContainerHeader>,
    );

    expect(
      testHeader.container.firstChild,
    ).toMatchSnapshot();
  });
});
