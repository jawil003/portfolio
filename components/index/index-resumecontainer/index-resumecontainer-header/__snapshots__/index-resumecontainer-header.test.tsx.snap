// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IndexResumeContainerHeader Create IndexResumeContainerHeader 1`] = `
.emotion-0 {
  text-align: start;
  font-style: italic;
  margin: unset;
  -webkit-text-decoration: none;
  text-decoration: none;
  letter-spacing: normal;
  word-spacing: normal;
  color: #000;
  word-wrap: unset;
  padding: unset;
  word-break: normal;
}

.emotion-0 {
  font-weight: 600;
  font-size: 2.125em;
}

<h4
  class="emotion-0 emotion-1"
  color="#000"
  id="heading4"
>
  Hello World
</h4>
`;
